### ISSUE TITLE
set up issue templates

### ISSUE THREAD

**Issue by jeremybmerrill** [tabulapdf/tabula#465]
We should set up Issue Templates for this repo. https://github.com/blog/2111-issue-and-pull-request-templates

We should include, at the very least,

- tabula version
- link to PDF, attached PDF or a promise to email it to us if it's private
- platform

thoughts?

**Comment by jazzido** [comment]
Yup. Also, locale of the platform. We get reports about path encoding issues from time to time.

**Comment by jeremybmerrill** [comment]
Yep, three today!

**Comment by jazzido** [comment]
A bunch of path encoding issues were resolved in JRuby since 1.7.20, but somewhere along the chain of build steps is using `1.7.15` ([see this](https://github.com/tabulapdf/tabula/issues/462#issue-134013359)).

**Comment by jeremybmerrill** [comment]
k pienses, @mtigas? Anything else we should add?
